README.md

```diff
@@ -119,7 +119,20 @@ A list of full environment variable settings is below.  They can be either manua
 Multitenancy is supported through postgres schemas.  Multiple schemas are specified using `SURVEY_SERVICE_DB_SCHEMA` as a '~' delimited string of schema names.  This project assumes that each schema has the same table structure during database synchronization.  Schema names are appended to the base url for each API end point so that each tenant can be accessed using a different path.
 
 ## Tests
-`yarn test`
+```sh
+# Make sure .env.test exists
```

Comment:
Do we want to include a `.env.test` and/or `.env.production` in the code base as is in the Messaging Service? Or do we want to remove the ones in the Messaging Service? If the latter, do we want to either introduce a `.env.test.example` of, if there's no difference here other than a change in the `NODE_ENV`, then perhaps make a note here in the README to create `.env.test` and/or `.env.production` when copying for the regular `.env`?

Reply:
@unprofessional : You're correct that `.env.test` should be included. Included in a separate PR, https://github.com/amida-tech/amida-survey-microservice/pull/71